sql/src/test/java/io/crate/integrationtests/JoinIntegrationTest.java

```diff
@@ -43,6 +48,14 @@
 @ESIntegTestCase.ClusterScope(minNumDataNodes = 2)
 public class JoinIntegrationTest extends SQLTransportIntegrationTest {
 
+    @Override
+    protected Settings nodeSettings(int nodeOrdinal) {
+        return Settings.builder()
+            .put(super.nodeSettings(nodeOrdinal))
+            .put("stats.service.interval", "2s")
```

Comment:
Why so big interval here? you can do it even ms I think

Reply:
The cluster is not ready and it just spams the logs with errors

Reply:
Would it be an option to just get the instance from the cluster state and fill in the stats manually within the tests?

Reply:
That's a great idea! Giving it a try now